task4/src/main/scala/CsvTask.scala

```diff
@@ -0,0 +1,122 @@
+
+import java.io._
+import java.text.SimpleDateFormat
+
+import scala.language.reflectiveCalls
+import scala.collection.mutable.{ArrayBuffer, Map}
+
+import java.time.format.DateTimeFormatter
+
+
+
+object CsvTask extends  App {
+
+  using(io.Source.fromFile("201608-citibike-tripdata.csv")) { source =>
+
+  {
+    var alltrevel = 0
+    var allbikes = Set[String]()
+    var bike = Map[String, Int ]()
+    var Manth = Map[Int, Int]()
+
+    var x = 1
+    val datetime_format = DateTimeFormatter.ofPattern("\"MM/dd/yyyy HH:mm:ss\"")
+    var date = 0
+
+
+
+
+    var mail  = 0
+    var femail = 0
+
+
+
+    val allstring  = source.getLines().toArray
+
+    for (line <- 1 to allstring.size-1) {
+      val split_string = allstring(line).split(",")
+
+
+      val format = new SimpleDateFormat("\"MM/dd/yyyy hh:mm:ss\"")
+      val date1 = (format.parse(split_string(1)))
+
+      val date2 = (format.parse(split_string(2)))
+
+
+      if (date<(((date2.getMinutes - date1.getMinutes)*60)+(date2.getSeconds - date1.getSeconds))){
```

Comment:
What are you trying to achieve  by this condition?
What (((date2.getMinutes - date1.getMinutes)*60)+(date2.getSeconds - date1.getSeconds))) means?